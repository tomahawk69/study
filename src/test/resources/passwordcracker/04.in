4
6
zsnpgbqh bktrpgdwbu qhuhzxfh mxrgmga omgtgnqomb dffttrwlfh
nktrsgtwbuzsbptzahomgtgnaoma
6
xkof medbc mhyewjzsdg vkuzym tbeqv xtbyhn
xtbyhnmedbcmhyewjzsdgxtbyhn
6
alutwfal kkhbqlrxnm qyyx lwdgpchwic rdtgzvw sduh
sduhkkhbqlrxnmsduhsduhqyyx
10
a aa aaa aaaa aaaaa aaaaaa aaaaaaa aaaaaaaa aaaaaaaaa aaaaaaaaaa
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaab