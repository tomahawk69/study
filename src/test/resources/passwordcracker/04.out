WRONG PASSWORD
xtbyhn medbc mhyewjzsdg xtbyhn
sduh kkhbqlrxnm sduh sduh qyyx
WRONG PASSWORD